webrtc.html

```diff
@@ -8713,6 +8713,15 @@ <h2 id="sec.cert-mgmt">Certificate Management</h2>
           values in <var>configuration</var>.</p>
         </li>
         <li>
+          <p>Initialize <var>channel</var>'s
+          <a data-for="RTCDataChannel"><code>priority</code></a> attribute to
+          the <a>RTCDataChannelPriority</a> enum value that corresponds to the
+          integer priority value in <var>configuration</var>, as defined in
```

Comment:
Oh, my mistake. Hmm. This is a problem then, because the implementation is allowed to use values other than 128/256/512/1024. What should the attribute return if the data channel was opened with a priority of 200? Maybe the `priority` attribute should be an integer, while the `RTCDataChannelInit` member remains an enum. Seems like a good thing to bring up in the upcoming virtual interim.